content/3_Silver/Two_Pointers.mdx

```diff
@@ -274,7 +274,41 @@ public class Books {
 ```
 
 </JavaSection>
-
+<PySection>
+
+```py
+n = int(input())
+t = int(input())
+
+books = [0] * n
+
+for i in range(n):
+	books[i] = int(input())
```

Comment:
probably better as:

```
books = list(map(int, input().split()))
```

Reply:
Aren't the inputs in multiple lines? I changed it to books = [int(input()) for _ in range(n)]. sorry if I'm wrong.

Reply:
oops sorry, you're right. thanks!